core/src/main/java/module-info.java

```diff
@@ -187,6 +187,7 @@
             io.questdb.griffin.engine.functions.math.AbsLongFunctionFactory,
             io.questdb.griffin.engine.functions.math.AbsDoubleFunctionFactory,
             io.questdb.griffin.engine.functions.math.LogDoubleFunctionFactory,
+            io.questdb.griffin.engine.functions.math.Log10DoubleFunctionFactory,
```

Comment:
functions have to be added to two places to maintain JDK8 compatibility:

- core/src/main/java/module-info.java
- core/src/main/resources/META-INF/services/io.questdb.griffin.FunctionFactory

Reply:
ok